docs/ResourceAuthorization.md

```diff
@@ -0,0 +1,172 @@
+# Resource Authorization
+
+Nakadi allows users to restrict access to resources they own - currently, event types are the only resources supported, 
+but we plan to extend this feature to subscriptions in the near future.
+
+The authorization model is simple: policies can be attached to resources. A policy `P` defines which `subjects` can 
+perform which `operations` on a resource `R`. To do so, the policy contains, for each operation, a list of attributes that 
+represent subjects who are authorized to perform the operation on the resource. For a subject to be authorized, it needs 
+to match at least one of these attributes (not necessarily all of them).
+
+There are three kinds of operation: `admin`, to update the resource and delete it; `read`, to read events from the 
+resource; and `write`, to write events to the resource.
+
+An authorization request is represented by the tuple
+
+`R(subject, operation, resource)`
+
+The request will be approved iff the `resource` policy has at least one attribute for `operation` that matches the 
```

Comment:
No :)

`iff` means `if and only if`: https://en.wikipedia.org/wiki/If_and_only_if